smartsim/_core/config/config.py

```diff
@@ -204,6 +204,17 @@ def test_account(self) -> t.Optional[str]:  # pragma: no cover
         # no account by default
         return os.environ.get("SMARTSIM_TEST_ACCOUNT", None)
 
+    @property
+    def telemetry_frequency(self) -> int:
+        return int(os.environ.get("SMARTSIM_TELEMETRY_FREQUENCY", 5))
+
+    @property
+    def telemetry_enabled(self) -> bool:
+        return bool(os.environ.get("SMARTSIM_FLAG_TELEMETRY", 1))
```

Comment:
If a user were to explicitly
```sh
export SMARTSIM_FLAG_TELEMETRY=0
```
then ``os.environ.get("SMARTSIM_FLAG_TELEMETRY", 1)`` would eval `"0"` (as a str type) and `bool("0")` would eval ``True``, when the user almost certainly intended for the TM to be off and for this to eval ``False``. We should make an explicit cast to `int` before casting to `bool`